lib/RefBackend/RefBackend.cpp

```diff
@@ -194,17 +194,17 @@ void mlir::NPCOMP::createRefBackendLoweringPipeline(
   //
   // Also, converting to linalg herevopens up a lot of optimization
   // opportunities.
-  pm.addPass(createConvertElementwiseToLinalgPass());
+  pm.addNestedPass<FuncOp>(createConvertElementwiseToLinalgPass());
```

Comment:
Thanks for making this explicit! Could you try giving a stab at removing OpPassManager::Nesting::Implicit from our codebase so we don't regress? I think this was the only offender. (if that doesn't "just work", then don't worry about it)

Reply:
I think this will be done by https://reviews.llvm.org/D91233